models/migrations/v62.go

```diff
@@ -112,5 +112,10 @@ func addMultipleAssignees(x *xorm.Engine) error {
 		}
 	}
 
-	return sess.Commit()
+	err = sess.Commit()
+	if err != nil {
+		return err
+	}
+
+	return dropTableColumns(x, "issue", "assignee_id")
```

Comment:
why not 
```Go
if err := dropTableColumns(sess, "issue", "assignee_id"); err != nil {
    return err
}
return sess.Commit()
```

Reply:
Because `models/migrations/v62.go:115:28: cannot use sess (type *xorm.Session) as type *xorm.Engine in argument to dropTableColumns`

Reply:
You can change `dropTableColumns(*xorm.Engine, string, string)` -> `dropTableColumns(Engine, string, string)` simply to resolve it.

Reply:
I'm sorry, I don't understood you. Do you mean changing the function definition?

Reply:
@kolaente Yes, that should be a slight change and should be compatible with other call places.

Reply:
Modified, I had to modify only one other migration.